### ISSUE TITLE
Plugin apply should be after android extension setup

### ISSUE THREAD

**Issue by CherryPerry** [jeremymailen/kotlinter-gradle#64]
This setup does not work, because source set is edited after plugin apply. Lint task just ends with `NO_SOURCE` result.
```groovy
apply plugin: 'org.jmailen.kotlinter'
android {
    sourceSets {
        main.java.srcDirs += 'src/main/kotlin'
    }
}
```

And this setup works well.
```groovy
android {
    sourceSets {
        main.java.srcDirs += 'src/main/kotlin'
    }
}
apply plugin: 'org.jmailen.kotlinter'
```

This is not a bug, because plugin creates tasks right after apply call. But I think, the second configuration example must be documented in README.md, because it can be not obvious enough for new users of plugin (like me), why it does not work.

**Comment by jeremymailen** [comment]
Thanks for the report! I actually do think it is a bug since gradle provides some lifecycle hooks to do this correctly. The root cause was reported in #53. There's actually a fix out for it awaiting final approval: #55

**Comment by jeremymailen** [comment]
Fixed in `1.15.0`
